### PR TITLE
doc: fill README.md

### LINKED ISSUE #5: docs: add README.md
We need a useful README.md with a short instruction how to use the role. It could be implemented based on some module/role (expirationd/etcd-client/crud etcd).

### PR DESCRIPTION
**Pull request by oleg-jukovec**
Closes #5

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,145 @@
 # metrics-export-role
-The Tarantool 3 role for metrics export via HTTP
+
+`roles.metrics-export` is a role for Tarantool 3. It allows to export metrics
+from Tarantool 3. At now only export via HTTP is supported.
```
Comment (DifferentialOrange) [inline README.md]: "At the moment" or "For now".

File: README.md [hunk 2]
```diff
@@ -1,2 +1,145 @@
...
+
+```Lua
+dependencies = {
+    ...
+    'metrics-export-role == 0.1.0-1',
+    ...
+}
+```
+
+And re-build the rockspec:
```
Comment (DifferentialOrange) [inline README.md]:
```suggestion
And re-build the application:
```

File: README.md [hunk 3]
```diff
@@ -1,2 +1,145 @@
...
+
+The role configuration at the top level could be described as:
+
+```yaml
+export_target: opts
+```
+
+### http target
+
+At now only `http` target is supported. The target allows to export metrics via
```
Comment (DifferentialOrange) [inline README.md]: Same

File: README.md [hunk 4]
```diff
@@ -1,2 +1,145 @@
...
+cd metrics-export-role
+```
+
+After that you need to install dependencies (`tt` is required):
+
+```shell
+make deps
+```
+
+At now you could run tests (`tarantool` 3 is required):
```
Comment (DifferentialOrange) [inline README.md]: Same

File: README.md (hunk 1 above)
Comment (oleg-jukovec) [reply README.md]: Thank you, `s/At now/For now/g`.

File: README.md (hunk 2 above)
Comment (oleg-jukovec) [reply README.md]: Thank you, updated.

File: README.md (hunk 3 above)
Comment (oleg-jukovec) [reply README.md]: Thank you, `s/At now/For now/g`.

**Review by DifferentialOrange — APPROVED** [review]

File: README.md (hunk 4 above)
Comment (oleg-jukovec) [reply README.md]: Thank you, `s/At now/At this point/g`.

**Comment by oleg-jukovec** [comment]
@better0fdead rebased

**Review by better0fdead — APPROVED** [review]
Hi, thx for patch

**Pull request merged**